### DISCUSSION TITLE
Design reviews

### DISCUSSION THREAD

**Discussion by kalapi** [universalthirst/noto-bengali#1] [General]
@joanaranito Using the 'Discussions' panel/feature for the first time to share review. Let's see how it goes. 

I've marked up the first 4 pages today. I'll go through the rest tomorrow. 

Here are my notes:

#### General
The loops in the Black feel a bit sqarish

#### Page 1

- [x] [aa-beng] Is the curve/notch too quick/small compared to the other weights?
- [x] [ii-beng] Is the bowl too small compared to the other weights?
- [x] [uu-beng] Should the inktrap extend further, atleast until the stroke is the max. possible width?
- [x] [r/rrVocalic-beng] The angle of the short diagonal stroke is too vertical copared to the other weights
- [x] [o/au-beng] Curve quality and maybe an inktrap at the junction (circle)?
- [ ] [gha-beng] Too little shirorekha on the left?
- [x] [ca/cha-beng] Curve quality
- [x] [ja-beng] Curve quality
- [x] [ka/jha-beng] The proportions of the 'ba'-form feel better in the /jha compared to the /ka

#### Page 2

- [x] [tha-beng] The downward curve in the Black expands instead of tapering like in the Regular. Which one is preferred?
- [x] [da-beng] Angle of the heavy diagonal in the Black is steep, could maybe be less so?
- [x] [dha-beng] The stroke-weight in the top-leftcurve could be a bit lighter?
- [x] [ba/ra-beng] The median tip in the /ba feels higher such that it extends the bottom stroke quite a bit. I know we don't want it to be an equilateral triangle, but better to be optically the same lengths?
- [x] [ma-beng] Curve quality
- [x] [ma-beng] Curve quality

#### Page 3

- [x] [raDiagonal-beng] The angle of the independant diagonal stroke could be fixed by fixing the /ba
- [x] [khandaTa-beng] Could we use a similar approach in thinning strokes or should we fix it here to be more equal?
- [x] [iiMatra-beng] I know we have limited space, but this feels too vertically compressed and the modulation is too extreme.
- [x] [aiMatra-beng] The matra looks a bit too 'vertical' as compared to the other weights which have a more generous curve/bulge.

#### Page 4

- [x] [l/lVocalicMatra-beng] Minor fixes to angles and curves as sketched.
- [x] [numerals] Do the knots look a bit informal in these glyphs – especially 1 & 7?

[2024-05-21_NotoBengali_UnicodeReview.pdf](https://github.com/universalthirst/noto-bengali/files/15395892/2024-05-21_NotoBengali_UnicodeReview.pdf)

*Edited 22nd May '24: Wrong page numbers for the last two issues*

**Comment by joanaranito** [comment]
Hi Kalapi, thank you for your initial review! I have some comments:

>  [aa-beng] Is the curve/notch too quick/small compared to the other weights?

It is small, but there's very little space for the curve when you take into account the stroke weight, the spacing, and the shirorekha overlap. To increase the curve, we'll have to steal from the top part of the stem, but we also can't steal too much or it will become too weak.

>  [ii-beng] Is the bowl too small compared to the other weights?

Yes, I agree. I was trying to keep the height difference between the two bottom elements, but I will bring both down to increase the size of the bowl.

>  [r/rrVocalic-beng] The angle of the short diagonal stroke is too vertical compared to the other weights

I'll make the other weights more vertical too. In the black, there isn't a lot of space for that stroke to extend without pushing the knot too far and while giving space for the downward stroke to register. So I'll find an intermediate angle that works for all weights.

>  [gha-beng] Too little shirorekha on the left?

Will have to check the spacing :) 

> [tha-beng] The downward curve in the Black expands instead of tapering like in the Regular. Which one is preferred?

It should either remain parallel or expand, because that's where the weight is. I don't see the tapering in the Regular, but I'll balance both out a bit better.

>  [ba/ra-beng] The median tip in the /ba feels higher such that it extends the bottom stroke quite a bit. I know we don't want it to be an equilateral triangle, but better to be optically the same lengths?

Since this is a supposed to match a humanist Latin rather than a geometric Latin, I think the higher median tip makes more sense. It's very common in more traditional Bangla styles. Have a look at page 50 where it's being compared to the Serif - in the Serif, the tip is even higher. I think it's part of the Noto's character, I wouldn't make the two strokes optically the same. I'd prefer to keep it as is.

>  [raDiagonal-beng] The angle of the independant diagonal stroke could be fixed by fixing the /ba

I don't necessarily think the angle has to be the same across weights, but I'll try out a few things to see if they can be a bit more harmonious. Note: because the Black is slightly wider than the other masters, to make sure that the counters still register, it's normal for those types of angles to suffer a bit of deformation. You can see the same happening in the Serif.

>  [khandaTa-beng] Could we use a similar approach in thinning strokes or should we fix it here to be more equal?

I don't think I understand what you mean 😅  Can you rephrase?

>  [iiMatra-beng] I know we have limited space, but this feels too vertically compressed and the modulation is too extreme.

If we're keeping to the original metrics, I think that there is no more room to extend the stroke, but I'll double check. To open it up we might need to lower the shirorekha back down a touch. Maybe we can change the design to the version that doesn't reattach to the shirorekha? I'll see what I can do. 

EDIT: Just to confirm that the winAscent is 917, and that the iiMatra goes all the way up to 917. But because the Black is currently too heavy a match for the Latin (because we've added knots and made the shapes narrower to better match the other masters), and because we're going to have to interpolate a lighter match, we're going to gain some space there, which will help improving the shape.

>  [aiMatra-beng] The matra looks a bit too 'vertical' as compared to the other weights which have a more generous curve/bulge.

Can you specify which weight this comment is for? 

> [l/lVocalicMatra-beng] Minor fixes to angles and curves as sketched.

I don't see any markings/sketches on the PDF, can you share them?

> [numerals] Do the knots look a bit informal in these glyphs – especially 1 & 7?

Yes, I agree, especially in the Black they look too "cute". I'll have a look.

**Reply by kalapi** [reply]
> > [khandaTa-beng] Could we use a similar approach in thinning strokes or should we fix it here to be more equal?

> I don't think I understand what you mean 😅 Can you rephrase?

Sure! In the loop of the KhandaTa there's a thin stroke merging into a thick stroke following the rules of the modulation. Shall we do that in other glyphs where this would occur to keep the logic same? Or should we modify the KhandaTa by decreasing the contrast? Here's an example with /La

![Screenshot 2024-05-22 at 9 47 25 AM](https://github.com/universalthirst/noto-bengali/assets/15804460/2e139c92-a760-44b2-91a2-c4777202e519)

> > [iiMatra-beng] I know we have limited space, but this feels too vertically compressed and the modulation is too extreme.

Can we try something like this in the Black where (similar to the shaping in the Serif) the thin bits of the letterform balance the heavy bits? Also, for the lower stroke we thin it a bit to follow the logic of low-contrast design principals thereby maintaining both the classical modulation but balanced shape overall? (Note: rough sketch, please don't judge quality 😬)

![Screenshot 2024-05-22 at 9 59 01 AM](https://github.com/universalthirst/noto-bengali/assets/15804460/49840192-f4f2-4ba5-91c2-cc6becf754ad)

>> [aiMatra-beng] The matra looks a bit too 'vertical' as compared to the other weights which have a more generous curve/bulge.

> Can you specify which weight this comment is for?

Sorry! It's for the Black. It affects: `/evowelsignbeng /aivowelsignbeng /ovowelsignbeng /auvowelsignbeng`

>>[l/lVocalicMatra-beng] Minor fixes to angles and curves as sketched.

>I don't see any markings/sketches on the PDF, can you share them?

Sorry again! This and the numerals comment is for page 4 (I've updated the original post). The markings are lighter so you may have to zoom in.

**Reply by joanaranito** [reply]
> Can we try something like this in the Black where (similar to the shaping in the Serif) the thin bits of the letterform balance the heavy bits? Also, for the lower stroke we thin it a bit to follow the logic of low-contrast design principals thereby maintaining both the classical modulation but balanced shape overall? (Note: rough sketch, please don't judge quality 😬)

Sure, let's try that! But I'll still think it makes sense to wait until we interpolate the black, otherwise we'll be making optical adjustments that won't apply to the same extent :) 

> Sure! In the loop of the KhandaTa there's a thin stroke merging into a thick stroke following the rules of the modulation. Shall we do that in other glyphs where this would occur to keep the logic same? Or should we modify the KhandaTa by decreasing the contrast? Here's an example with /La

If the modulation of the khandaTa is too much, we should decrease it there, rather than increase it everywhere else. I was trying to follow the same shaping as in the Serif, but maybe it needs to connect lower to keep the modulation low.
